### DISCUSSION TITLE
FIS Buyer APP or STUB API for testing

### DISCUSSION THREAD

**Discussion by nemishzalavadiya-pirimid** [ONDC-Official/ONDC-FIS-Specifications#50] [Q&A]
Please share resource to get FIS Buyer APP for testing

**Comment by nemishzalavadiya-pirimid** [comment]
Please share how to test and where to get STUB api as discussed in last call for testing after subscription.

**Reply by 92shreyansh** [reply]
We are currently working on STUB creation will make it available soon

**Reply by 92shreyansh** [reply]
Time being lets bring this up in next community call where if any Buyer App has already in the phase of development and on staging can support this for time being
